src/theme/kubera-core.tsx

```diff
@@ -87,36 +87,40 @@ const kuberaCoreTheme = createTheme({
   kuberaCoreCards: {
     kuberaChaos: {
       background: {
-        default: 'linear-gradient(268.56deg, #443F93 1.65%, #292A5B 61.02%)',
-        paper: 'linear-gradient(255.08deg, #0F7B5B 7.37%, #0E5B44 100%)',
+        default: 'linear-gradient(267.51deg, #118F6A 1.47%, #0C262E 100%)',
+        paper:
+          'linear-gradient(255.08deg, #10805E 7.37%, rgba(16, 128, 94, 0) 100%)',
       },
       button: {
-        unconfigured: '#52F99540',
-        configured: '#2BCB69',
+        unconfigured: 'rgba(82, 249, 149, 0.25)',
```

Comment:
can you use hex codes just for consistency

Reply:
👍🏻